src/PowerShellEditorServices/Extensions/BuiltInCommands.ps1

```diff
@@ -0,0 +1,29 @@
+Register-EditorCommand `
+    -Name 'PowerShellEditorServices.OpenEditorProfile' `
+    -DisplayName 'Open Editor Profile' `
+    -SuppressOutput `
+    -ScriptBlock {
+        param([Microsoft.PowerShell.EditorServices.Extensions.EditorContext]$context)
+        If (!(Test-Path -Path $Profile)) { New-Item -Path $Profile -ItemType File }
+        $psEditor.Workspace.OpenFile($Profile)
+    }
+
+Register-EditorCommand `
+    -Name 'PowerShellEditorServices.OpenProfileList' `
+    -DisplayName 'Open Profile from List (Current User)' `
+    -SuppressOutput `
+    -ScriptBlock {
+        param([Microsoft.PowerShell.EditorServices.Extensions.EditorContext]$context)
+        
+        $List = @('Microsoft.VSCode_profile.ps1','Microsoft.PowerShell_profile.ps1','Microsoft.PowerShellISE_profile.ps1','Profile.ps1')
```

Comment:
I missed this part when I looked at this earlier, we should use the filename of the current editor's profile as the first item rather than hardcoding the VS Code option.  Makes sense to hardcode the other profiles though since they are common on Windows.